automation/submission-perf-test.sh

```diff
@@ -40,27 +77,25 @@ launchSubmission() {
         expression=$9
     "
 
-    ACCESS_TOKEN=`docker run --rm -v $WORKING_DIR:/app/populate -w /app/populate broadinstitute/dsp-toolbox python get_bearer_token.py "${user}" "${JSON_CREDS}"`
-
-    # Verify that user does not need to refresh their token
-    if curl -f -v --silent -X GET --header "Accept: application/json" --header "Authorization: Bearer $ACCESS_TOKEN" "https://firecloud-orchestration.dsde-alpha.broadinstitute.org/api/refresh-token-status"  2>&1 | grep '"requiresRefresh": true'
-    then
-        echo "This user needs its refresh token refreshed"
-        exit 1
-    fi
-
-    echo "Note: If there is an error message below (with 'curl: (22)' and '404 Not Found') then that user needs to re-link to NIH."
+     ACCESS_TOKEN=`docker run --rm -v $WORKING_DIR:/app/populate -w /app/populate broadinstitute/dsp-toolbox python get_bearer_token.py "${user}" "${JSON_CREDS}"`
 
-    #check if $9 is set for 'expression'
+    # check if $9 is set for expression
     if [ -z ${9+x} ] ; then
         curl -f "https://firecloud-orchestration.dsde-$ENV.broadinstitute.org/api/workspaces/$namespace/$name/submissions" -H "origin: https://firecloud.dsde-$ENV.broadinstitute.org" -H "accept-encoding: gzip, deflate, br" -H "authorization: Bearer $ACCESS_TOKEN" -H "content-type: application/json" --data-binary "{\"methodConfigurationNamespace\":\"$methodConfigurationNamespace\",\"methodConfigurationName\":\"$methodConfigurationName\",\"entityType\":\"$entityType\",\"entityName\":\"$entityName\",\"useCallCache\":$useCallCache}" --compressed
     else
         curl -f "https://firecloud-orchestration.dsde-$ENV.broadinstitute.org/api/workspaces/$namespace/$name/submissions" -H "origin: https://firecloud.dsde-$ENV.broadinstitute.org" -H "accept-encoding: gzip, deflate, br" -H "authorization: Bearer $ACCESS_TOKEN" -H "content-type: application/json" --data-binary "{\"methodConfigurationNamespace\":\"$methodConfigurationNamespace\",\"methodConfigurationName\":\"$methodConfigurationName\",\"entityType\":\"$entityType\",\"entityName\":\"$entityName\",\"useCallCache\":$useCallCache,\"expression\":\"$expression\"}" --compressed
     fi
 }
 
-
 if [ $ENV = "alpha" ]; then
+for user in "${users[@]}"
+    do
+        checkToken $user
+    done
+    for user in "${users[@]}"
+    do
+        callbackToNIH $user
```

Comment:
If we keep this implementation the way it is, you could put both checkToken and callbackToNIH into one for loop

Reply:
I don't want to do so, if user needs token refresh I want the script to stop without going further

Reply:
I think it would eliminate duplicate code. I think we can do both in one for loop.

Also, we can stick this check before ` if [ $ENV = "alpha" ]; then`. (Both alpha and staging use the same users so it just needs to be run once before either of them run. )

Also, indentation ;)